package.json

```diff
@@ -135,7 +135,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-helmet": "^6.1.0",
-    "react-lazyload": "^2.6.8",
+    "react-lazyload": "^3.0.0",
```

Comment:
@rhenshaw56 Are there any breaking changes in this major version bump?

Reply:
@joshcoventry I spotted no breaking changes. The snapshot updates was as a result of the vendor appending `classnames` to the component wrapper [see commit](https://github.com/twobin/react-lazyload/commit/0b042b5d7656f45ec44f91ac76122f2a32e11704)